### PR TITLE
add rest of packages to workspaces

### PR DESCRIPTION
**Pull request by dwalkr**

### CODE REVIEW

File: packages/@tinacms/toolkit/src/components/media/media-manager.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ limitations under the License.
 import React from 'react'
 import { useEffect, useState } from 'react'
 import styled, { css } from 'styled-components'
-import { useCMS } from '../../react-tinacms'
+import { useCMS } from '../../react-tinacms/use-cms'
```
Comment (Enigmatical) [inline packages/@tinacms/toolkit/src/components/media/media-manager.tsx:22]: Can you help me understand the difference between importing `useCMS` from `../../react-tinacms/use-cms` versus other places where we import from `tinacms`?

(Example being packages/react-tinacms-github/src/github-client/GithubFile.ts)

Comment (dwalkr) [reply packages/@tinacms/toolkit/src/components/media/media-manager.tsx:22]: this is to fix a circular dependency internally within this package. external packages will just be importing from `tinacms`.

the existence of both is likely vestigial at this point now that everything is merged, but that will need to wait for a future refactor.

**Review by Enigmatical — COMMENTED** [review]
Aside from my one question about the use of `useCMS()`, this looks good 👍 

I also appreciate the extra effort of cleaning up a few things too - breaking out `BlockAction` into its own component, `getOffset` becoming its own import, etc.  Very nice!

File: packages/@tinacms/toolkit/src/components/media/media-manager.tsx (hunk 1 above)
Comment (Enigmatical) [reply packages/@tinacms/toolkit/src/components/media/media-manager.tsx:22]: Ah, that makes sense. 👍

**Review by Enigmatical — APPROVED** [review]

**Pull request merged**